packages/utils/src/memorize/index.ts

```
@@ -0,0 +1,160 @@
+  sort(list) {
+    if (Array.isArray(list)) {
+      return list.sort((x, y) => {
+        const isDesc = this._sort === 'DESC'
+        const compare = isDesc ? [-1, 1] : [1, -1]
+
+        const sortField = this._sortBy === 'FREQUENCY' ? this._customField1 : this._customField2
+
+        const xField = x[sortField]
+        const yField = y[sortField]
+
+        if (isNaN(xField)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)